src/utils/utils.js

```diff
@@ -1,12 +1,17 @@
 export const searchUsers = (query, users) => {
-  return users.filter(user => user.name.toLowerCase().includes(query.toLowerCase()));
+  return users.filter(user =>
+    user.name.toLowerCase().includes(query.toLowerCase())
+  );
 };
 
-export const filterPosts = (posts, id) => {
-  return posts.filter(item => item.userId === parseInt(id));
+export const replaceID = (route, ...ids) => {
```

Comment:
Naming should follow the general convention of Camel Case. We can't capitalise one word randomly.